### PR TITLE
{tools}[foss/2021b] Arcade-Learning-Environment v0.7.3 w/ Python 3.9.6

### PR DESCRIPTION
**Pull request by lexming**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by lexming** [comment]
@boegelbot: please test @ generoso

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
node304.hydra.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/ed7eec81f33323f87b40dc50ecbee8d2 for a full test report.

**Comment by lexming** [comment]
@boegelbot: please test @ generoso

File: easybuild/easyconfigs/a/Arcade-Learning-Environment/Arcade-Learning-Environment-0.7.3-foss-2021b.eb [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+        'source_tmpl': 'v%(version)s.tar.gz',
+        'checksums': ['e3bada34cc6c116377c4a807c24d9890ce33afa854ffc45e32dc90ba0dcc9140'],
+        'preinstallopts': "ALE_BUILD_VERSION=%(version)s",
+    }),
+]
+
+sanity_check_paths = {
+    'files': ['bin/ale-import-roms', 'lib64/libale.a'],
+    'dirs': ['include/ale', 'lib/python%(pyshortver)s/site-packages/'],
+}
```
Comment (boegel) [inline easybuild/easyconfigs/a/Arcade-Learning-Environment/Arcade-Learning-Environment-0.7.3-foss-2021b.eb:53]: @lexming Let's add a sanity check command here too?

```suggestion
}

sanity_check_commands = ["ale-import-roms --help"]
```

**Review by boegel — CHANGES_REQUESTED** [review]

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3112.skitty.os - Linux UNKNOWN UNKNOWN, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/61eb145bd96c3692e20653a9d1310d09 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3508.doduo.os - Linux UNKNOWN UNKNOWN, x86_64, AMD EPYC 7552 48-Core Processor (zen2), Python 3.6.8
See https://gist.github.com/6e39b73ac8113deb4adefe8396d3770a for a full test report.

**Comment by boegel** [comment]
@boegelbot please test @ jsc-zen2

File: easybuild/easyconfigs/a/Arcade-Learning-Environment/Arcade-Learning-Environment-0.7.3-foss-2021b.eb (hunk 1 above)
Comment (lexming) [reply easybuild/easyconfigs/a/Arcade-Learning-Environment/Arcade-Learning-Environment-0.7.3-foss-2021b.eb:53]: good idea, thanks :+1:

**Review by boegel — APPROVED** [review]

**Comment by boegel** [comment]
Going in, thanks @lexming!

**Pull request merged**